src/main/java/org/sciborgs1155/robot/shooter/feeder/FeederIO.java

```diff
@@ -0,0 +1,5 @@
+package org.sciborgs1155.robot.shooter.feeder;
+
+public interface FeederIO extends AutoCloseable {
+  public void setVoltage(double voltage);
```

Comment:
for now, I don't think we will be using PID control for the feeder, meaning it's better to use `set` and keep our input to a duty cycle (-1 to 1) value because that method will provide voltage compensation that we'd otherwise get from a feedforward